app/Models/Game.php

```diff
@@ -86,39 +86,39 @@ class Game extends Model
     protected $guarded = [];
     protected $appends = ['votes'];
 
-    public function user(){
+    public function user() {
         return $this->hasOne('App\Models\User', 'id', 'user_id');
     }
 
-    public function maker(){
+    public function maker() {
         return $this->hasOne('App\Models\Maker', 'id', 'maker_id');
     }
 
-    public function developers(){
+    public function developers() {
         return $this->hasMany('App\Models\GamesDeveloper', 'game_id', 'id');
     }
 
-    public function screenshots(){
+    public function screenshots() {
         return $this->hasMany('App\Models\Screenshot');
     }
 
-    public function comments(){
+    public function comments() {
         return $this->hasMany('App\Models\Comment', 'content_id', 'id')->Where('content_type', '=', \DB::raw("'game'"))->with('user');
     }
 
-    public function getVotesAttribute(){
+    public function getVotesAttribute() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->